sqlpage/templates/shell.handlebars

```diff
@@ -49,76 +49,79 @@
-                                <a class="nav-link {{#if this.submenu}}dropdown-toggle{{/if}}" href="{{#if this.link}}{{this.link}}{{else}}#{{/if}}"
-                                    {{#if this.submenu}}data-bs-toggle="dropdown" data-bs-auto-close="outside"{{/if}}
-                                    role="button"
-                                >
-                                    {{this.title}}
-                                </a>
-                                {{#if this.submenu}}
-                                    <div class="dropdown-menu dropdown-menu-end" data-bs-popper="static">
-                                        {{#each this.submenu}}
-                                            <a class="dropdown-item" href="{{this.link}}">
-                                                {{this.title}}
-                                            </a>
-                                        {{/each}}
-                                    </div>
-                                {{/if}}
-                                </li>
-                            {{else}}
-                                <li class="nav-item">
-                                    <a class="nav-link text-capitalize" href="{{this}}.sql">{{this}}</a>
-                                </li>
-                            {{/if}}
-                        {{/each}}
-                    </ul>
-                    {{#if search_target}}
-                        <form class="d-flex" role="search" action="{{search_target}}">
-                            <input class="form-control me-2" type="search" placeholder="Search" aria-label="Search"
-                                   name="search" value="{{search_value}}">
-                            <button class="btn btn-outline-success" type="submit">Search</button>
-                        </form>
-                    {{/if}}
-                </div>
-            </div>
-        </nav>
+		<header id="sqlpage_header">
+			<nav class="navbar navbar-expand-md navbar-light">
+				<div class="container-fluid">
+					<a class="navbar-brand flex-grow-1 overflow-hidden" href="{{#if link}}{{link}}{{else}}/{{/if}}">
+						{{#if image}}
+							<img src="{{image}}" alt="{{title}}" width="32" height="32"
+								class="navbar-brand-image">
+						{{/if}}
+						{{#if icon}}
+							{{~icon_img icon~}}
+						{{/if}}
+						<h1 class="mb-0 w-0 fs-2">{{title}}</h1>
+					</a>
+					<button class="navbar-toggler" type="button" data-bs-toggle="collapse"
+						data-bs-target="#navbar-menu" aria-controls="navbar-menu" aria-expanded="false"
+						aria-label="Toggle navigation">
+						<span class="navbar-toggler-icon"></span>
+					</button>
+					<div class="collapse navbar-collapse" id="navbar-menu">
+						<ul class="navbar-nav ms-auto">
+							{{#each (to_array menu_item)}}
+								{{#if (eq (typeof this) 'object')}}
+									<li class="nav-item{{#if this.submenu}} dropdown{{/if}}">
```

Comment:
The identation change makes it hard to see the diff here

Reply:
Oh shoot, I'll fix that 😄 My editor is set to huge indents, and that must have slipped in here.